lib/sidekiq/processor.rb

```diff
@@ -37,7 +36,7 @@ def initialize(mgr)
       @thread = nil
       @strategy = (mgr.options[:fetch] || Sidekiq::BasicFetch).new(mgr.options)
       @reloader = Sidekiq.options[:reloader]
-      @logging = (mgr.options[:job_logger] || Sidekiq::JobLogger).new
+      @job_logger = (mgr.options[:job_logger] || Sidekiq::JobLogger).new
```

Comment:
Would like to stress here we gonna use the specific logger (actually wrapper around the generic logger).